samples/nrf_rpc/protocols_serialization/client/snippets/crash_gen/crash_gen.conf

```diff
@@ -0,0 +1,8 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+CONFIG_NRF_RPC_CRASH_GEN=y
```

Comment:
why is this needed as a snippet when it can just be set in Kconfig fragment file, same with other snippet

Reply:
We would like to keep a uniformed usage as we use snippets also with other features, e.g. coex. Also, more configuration values may be added in the future

Reply:
this is not what snippets were intended for tbh

Reply:
I have removed both snippets. Customer that requested this feature will just edit one's prj.conf

Reply:
It can be added as a file suffix or set as part of a different test in sample.yml to show how to use it